analysers/analyser_merge_power_plant_FR.py

```diff
@@ -20,12 +20,10 @@
 ##                                                                       ##
 ###########################################################################
 
-import re
 from io import open # In python3 only, this import is not required
 from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
 from .Analyser_Merge_Geocode_Addok_CSV import Geocode_Addok_CSV
-from .modules import downloader
-from .modules import Stablehash
+from modules import Stablehash
```

Comment:
Again on this one. Please can you remove this change from this commit. As there is others `from .modules` but untouched by this commit. We may work on this in an other PR. Thank.

Reply:
ok, I keep it as .modules. I still think this is wrong and can currently only guess why Python manges to resolve it.

Reply:
Just seen why python manages to resolve in the wrong location. There are symbolic links in place.

Reply:
I think, it is required for tests.